templates/management/context.tpl

```diff
@@ -44,8 +44,7 @@
 			{load_url_in_div id="sectionsGridContainer" url=$sectionsGridUrl}
 		</tab>
 		<tab id="categories" label="{translate key="grid.category.categories"}">
-			{capture assign=categoriesUrl}{url router=PKP\core\PKPApplication::ROUTE_COMPONENT component="grid.settings.category.CategoryCategoryGridHandler" op="fetchGrid" escape=false}{/capture}
-			{load_url_in_div id="categoriesContainer" url=$categoriesUrl}
+			<category-manager v-bind="pageInitConfig"></category-manager>
```

Comment:
Wasn't sure where `pageInitConfig` was coming from so just wanted to confirm this is correct.

Reply:
Yes, this is correct. `pageInitConfig` is defined [here](https://github.com/pkp/pkp-lib/blob/be2746e84692632a4d210ab1fd2ca99c3c79816e/pages/management/ManagementHandler.php#L187-L189)